### ISSUE TITLE
Can't start goenovim in windows

### ISSUE THREAD

**Issue by shiena** [akiyosi/goneovim#542]
- OS: Windows11
- goneovim: a1d35b85ac7c5c51338dde31c62b7768aa8dd0a6

![image](https://github.com/user-attachments/assets/5622065a-677a-4c2a-ba40-d0d5b03d4bc5)

Similarly, libmd4c.dll, libicuuc75.dll and libzstd.dll also display error dialogs.

**Comment by shiena** [comment]
Has this problem been fixed by #541 ?

**Comment by akiyosi** [comment]
@shiena 
Thanks for this issue report.
The impetus for merging the changes that originally caused the degression was that CI began to fail only on Windows. We are sorry that the fixes we implemented in response were too messy and generated incorrect binaries.

I went back to the problem and did a little research, and it seems that the latest qt5-static package has been rewritten([See](https://github.com/msys2/MINGW-packages/commit/45cd41147449c6f276f8fe613cbd6289d703007d)) so that LD cannot find the library in the goneovim build process and an error occurs. I have tried simply specifying the library path in CGO_LDFLAGS, but that does not solve the problem, so it seems that the qt bindings mechanism needs to be rethought, which may take some time and effort.

For this reason, I'm going to change to the method of directly installing the qt5-static binary from the version where CI was working well. We are currently testing to see if CI works well with this method.
